File: vb.net/get-senders.vb
Dim client As New RestClient("<REPLACE_WITH_PROD_URL>/v1/senders")

client.Timeout = -1

Dim request As New RestRequest(Method.GET)

request.AddHeader("Authorization", "Bearer <REPLACE_WITH_YOUR_API_KEY>")
request.AddHeader("Content-Type", "application/json")

request.AddQueryParameter("query", "searchTerm")        
request.AddQueryParameter("status", "PENDING")         
request.AddQueryParameter("allAccounts", "true")      

Dim response As IRestResponse = client.Execute(request)
Console.WriteLine(response.Content)

File: php/guzzle/get-senders.php
<?php
// GUZZLE
// Make sure to install Guzzle using Composer: composer require guzzlehttp/guzzle

use GuzzleHttp\Client;
use GuzzleHttp\Psr7\Request;

$prodUrl = '<REPLACE_WITH_PROD_URL>';
$token = '<REPLACE_WITH_YOUR_API_KEY>';

$client = new Client();
$headers = [
    'Authorization' => 'Bearer ' . $token,
    'Content-Type' => 'application/json'
];

$params = [
    'query' => 'searchTerm',
    'status' => 'PENDING',
    'allAccounts' => 'true'
];

$url = $prodUrl . '/v1/senders?' . http_build_query($params);
$request = new Request('GET', $url, $headers);
$res = $client->sendAsync($request)->wait();
echo $res->getBody();